src/utils.cpp

```diff
@@ -31,11 +31,11 @@
 #include <zypp/ZYppFactory.h>
 #include <zypp/ResPool.h>
 #include <zypp/PoolItemBest.h>
+#include <zypp/base/Easy.h>
 #include "utils.h"
 
 // Define static variables here
-RepoManagerOptions* ZypperUtils::s_repoManagerOpts = new RepoManagerOptions("/tmp");
-RepoManager* ZypperUtils::s_repoManager = new RepoManager(*s_repoManagerOpts);
+scoped_ptr<RepoManager> ZypperUtils::s_repoManager( new RepoManager( Pathname( "/tmp" ) ) );
```

Comment:
Next question of course: Why ```/tmp```?
The ```RepoManager``` CTOR will scan the directory for .repo files to parse. So using ```/tmp``` is not a good choice as you know liittle about it's content. If you actually have temp. .repo files you want to use here, you may consider using a subdirectory to be able to control the content.

If it's just a _dummy_ until you know how to actually initialize the _real_ ```RepoManager```, then you should leave ```s_repoManager``` default-initialized here (i.e. ```== nullptr```) to avoid unnecessary/unwanted parsing. Furthermore a friendly exception ;) will tell you, if you access the RM before it's initialized it with valid data.